src/System.Security.SecureString/src/System/Security/SecureString.Windows.cs

```diff
@@ -103,9 +103,15 @@ private unsafe void InsertAtCore(int index, char c)
             }
             finally
             {
-                ProtectMemory(decryptedBuffer);
-                if (bufferPtr != null)
-                    decryptedBuffer.ReleasePointer();
+                try
+                {
+                    if (bufferPtr != null)
+                        decryptedBuffer.ReleasePointer();
```

Comment:
I was probably just being overly protective.  It could fail if decryptedBuffer was already disposed, but then the ProtectMemory call would also fail.  It could fail if decryptedBuffer had elsewhere had ReleasePointer erroneously called to decrement the value, but that would indicate a bug elsewhere in the SecureString implementation.  I'll remove the extra try/finally.